db/migrate/20201102200130_create_bookings.rb

```diff
@@ -0,0 +1,9 @@
+class CreateBookings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bookings do |t|
+      t.string :requested_booking_fields
+
```

Comment:
Here we should have:

```suggestion
      t.references :team, null: false, foreign_key: true
      t.references :user, null: false, foreign_key: true
      t.references :tournament, null: false, foreign_key: true
```